test/e2e_node/dra_test.go

```diff
@@ -178,7 +178,7 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 			ginkgo.By("wait for NodePrepareResources call to fail")
 			gomega.Eventually(kubeletPlugin.GetGRPCCalls).WithTimeout(retryTestTimeout).Should(testdriver.NodePrepareResourcesFailed)
 
-			unset()
+			unsetNodePrepareResourcesFailureMode()
```

Comment:
does this needs to be in defer as well? In case test will fail in the middle?

Reply:
As kubeletPlugin is created for every test case it's safe enough not to defer in my opinion.